purchase_discount/models/account_invoice_line.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import api, models
+
+class AccountInvoiceLine(models.Model):
+    _inherit = "account.invoice.line"
+
+    @api.model
+    def new(self, values={}):
+        """
+        Apply the linked to a purchase.order.line.discount to the
+        account_invoice_line
+        """
+        account_invoice_line = super(
```

Comment:
I didn't found a different way to pass it...mayebe you can explain me how this could be a problems? And if you think about a better place to add this discount into the account_invoice_line? Thanks!